### ISSUE TITLE
Getting started: Sigma vs Linkurious and other resources?

### ISSUE THREAD

**Issue by nixsee** [jacomyal/sigma.js#801]
Hi folks,

I'm sorry that this is such a basic question, but I hope it can help myself and other non-developers learn to use this incredible tool. I understand the basics of html, css, javascript and will certainly be working to get better at it all, but I hope you can point me/others in the right direction.

Anyway, I'm wondering what the difference is between the Sigma and Linkurious.  I know Linkurious is built on sigma, but not sure where the difference lies.

- Is there a recommendation for which to go with before I get started?
- Can they both do Canvas, WebGL and SVG?
- Can plugins for one can be used in the other? 
- Does one have more plans to be updated in the future than the other? It seems like the folks at Linkurious have moved on to other commercial projects 
- I was planning to use Gephi to style it and export a JSON, but is it better to simply import raw data and then create the entire chart programmatically?
- Finally, are there any recommendations of specific concepts to brush up on as well as resources/tutorials that might be helpful?

Thanks so much.

**Comment by dunnock** [comment]
Hi @nixsee, Linkurious is a fork of sigma and it is not maintained anymore as it is stated on their repo, so I would not start with that. I have used at least few Linkurious plugins with Sigma 1.2. In regards to format you can export to GEXF and use plugins.parsers.GEXF to load it. I suggest to look under examples, plugins and wiki, it helped me to quickly grasp over this lib.

**Comment by ekkis** [comment]
I'm wondering, if Linkurious is a fork of this project, couldn't we merge the changes in and continue development on this project?

**Comment by sim51** [comment]
There is some licence concerns ... (MIT vs GPL)

**Comment by ekkis** [comment]
that'a a real pity.  I'm guessing that means that whatever functionality was developed in Linkurious would have to be re-developed here? or is there a way to do an analysis of the licensing conflicts?

**Comment by ekkis** [comment]
yes, I'm not very happy about it.  I have a dependency.  I guess for now I'll live with it but down the road I might have to invest in sigma, to bring in whatever functionality is missing from linkurious
